common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -113,9 +113,13 @@ def textline_input_xml(**kwargs):
         """
         math_display = kwargs.get('math_display', False)
         size = kwargs.get('size', None)
+        input_element_label = kwargs.get('input_element_label', '')
```

Comment:
For the problem that has multiple text input fields, the question text is for the whole problem, but each text input also needs its own label. This is consistent with our example template.